### PR TITLE
docs: add more get started topics

### PR DESCRIPTION
**Pull request by fzn0x**
Remade from #101

### CODE REVIEW

**Comment by fzn0x** [comment]
This PR also fixed the CSS issue reported by @WikiRik

File: src/components/homepage-features.tsx [hunk 1]
```diff
@@ -28,7 +28,6 @@ const FeatureList: FeatureItem[] = [
         title: DataTypes.STRING,
         quantity: DataTypes.NUMBER,
       });
-
```
Comment (WikiRik) [inline src/components/homepage-features.tsx:31]: Can you revert the changes in this file? I think the empty lines makes the examples easier to understand

File: src/components/homepage-features.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const FeatureList: FeatureItem[] = [
...
+
```
Comment (ephys) [inline src/components/homepage-features.tsx:31]: Can you leave these lines empty? (no extra whitespace, just the new line) like it was before?

File: src/components/homepage-features.tsx [hunk 3]
```diff
@@ -103,7 +102,32 @@ export default function HomepageFeatures(): JSX.Element {
...
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/typescript/"
+          > strong typing
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/other-data-types/#json-sqlite-mysql-mariadb-and-postgresql-only"
+          > JSON querying
```
Comment (ephys) [inline src/components/homepage-features.tsx:123]: Keep the space outside of the link, otherwise it looks weird 
image

Comment (WikiRik) [reply src/components/homepage-features.tsx:123]: @fzn0x could you also fix this? I think we should be good to merge after that

**Comment by fzn0x** [comment]
Hi, done adding the whitespace outside the link @ephys @WikiRik

File: src/components/homepage-features.tsx [hunk 4]
```diff
@@ -103,7 +102,32 @@ export default function HomepageFeatures(): JSX.Element {
...
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/migrations/"
+          >migrations
+          </Link>,&nbsp;
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/typescript/"
+          >strong typing
+          </Link>,&nbsp;
```
Comment (ephys) [inline src/components/homepage-features.tsx:119]: Don't use non breaking spaces for these, use a regular space instead:

```suggestion
          </Link>,{' '}
```

**Review by ephys — CHANGES_REQUESTED** [review]

**Comment by fzn0x** [comment]
Hi, done adding the whitespace outside the link @ephys @WikiRik

File: src/components/homepage-features.tsx [hunk 5]
```diff
@@ -50,7 +50,6 @@ const FeatureList: FeatureItem[] = [
       const wish = await wishlist.createWish({ 
         title: 'Toys', quantity: 3,
       });
-
```
Comment (WikiRik) [inline src/components/homepage-features.tsx:53]: Can you undo this change so we keep the empty line?

**Comment by fzn0x** [comment]
Hi, done adding [keep the empty line](https://github.com/sequelize/website/pull/143/commits/6cebb897302589426059875d31c2e708f410637b) @WikiRik

**Comment by fzn0x** [comment]
@ephys If you have any review again please jump in, hope we can merge this faster ;)

**Comment by fzn0x** [comment]
@ephys If you have any review again kindly jump in, hope we can merge this faster ;)

**Review by WikiRik — APPROVED** [review]
Looks good to me!

**Review by ephys — APPROVED** [review]
This is good to go :)

**Comment by WikiRik** [comment]
Probably not due to this PR, but the production build of the v6 API reference is failing;

SyntaxError: Invalid regular expression: /[~][string, ..$/: Unterminated character class
    at new RegExp (<anonymous>)
    at IdentifiersDocBuilder._findByName (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:116:20)
    at IdentifiersDocBuilder._findByName (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:129:30)
    at IdentifiersDocBuilder._buildDocLinkHTML (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:782:22)
    at IdentifiersDocBuilder._buildTypeDocLinkHTML (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:764:19)
    at IdentifiersDocBuilder._buildSignatureHTML (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:849:27)
    at /home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:335:34
    at IceCap.loop (/home/runner/work/website/website/.sequelize/v6/node_modules/ice-cap/out/src/IceCap.js:261:9)
    at IdentifiersDocBuilder._buildSummaryDoc (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:331:9)
    at /home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/IdentifiersDocBuilder.js:75:28

It's re-running now, hopefully it's just a one off error

**Comment by ephys** [comment]
One of the recent additions to the v6 branch could have added a JSDoc type that esdoc does not like :/

**Comment by fzn0x** [comment]
> Probably not due to this PR, but the production build of the v6 API reference is failing;
> 
> ```
> SyntaxError: Invalid regular expression: /[~][string, ..$/: Unterminated character class
>     at new RegExp (<anonymous>)
>     at IdentifiersDocBuilder._findByName (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:116:20)
>     at IdentifiersDocBuilder._findByName (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:129:30)
>     at IdentifiersDocBuilder._buildDocLinkHTML (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:782:22)
>     at IdentifiersDocBuilder._buildTypeDocLinkHTML (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:764:19)
>     at IdentifiersDocBuilder._buildSignatureHTML (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:849:27)
>     at /home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:335:34
>     at IceCap.loop (/home/runner/work/website/website/.sequelize/v6/node_modules/ice-cap/out/src/IceCap.js:261:9)
>     at IdentifiersDocBuilder._buildSummaryDoc (/home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/DocBuilder.js:331:9)
>     at /home/runner/work/website/website/.sequelize/v6/node_modules/esdoc-publish-html-plugin/out/src/Builder/IdentifiersDocBuilder.js:75:28
> ```
> 
> It's re-running now, hopefully it's just a one off error

I thought it was from this PR xD

**Comment by fzn0x** [comment]
Thanks for the information anyway!
